beetsplug/fetchart.py

```diff
@@ -314,6 +314,16 @@ class CoverArtArchive(RemoteArtSource):
         GROUP_URL = 'http://coverartarchive.org/release-group/{mbid}/front'
 
     def get(self, album, plugin, paths):
+        """Cover Art Archive API offers pre-resized thumbnails at several sizes.
+        If the maxwidth config matches one of the already available sizes
+        fetch it directly intead of fetching the full sized image and resizing
+        it."""
+        VALID_THUMBNAIL_SIZES = [250, 500, 1200]
```

Comment:
`CAPITALIZED_NAMES` are generally only used for module-level constants. It's fine to make this lower-case, or to move it to a module-level constant with a more specific name.

Reply:
Thanks, I'm new with Python and beets so I'm learning as I go. I've already fixed this but didn't see your comment about the change log, so I'll do that as well now.